src/main/java/pl/zaprogramujzycie/mma/exceptions/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,72 @@
+package pl.zaprogramujzycie.mma.exceptions;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+import pl.zaprogramujzycie.mma.exceptions.ErrorMessage;
+
+import java.util.Date;
+
+@Slf4j
+@ControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
Zamiast rozszerzac klase, wystarczy @RestControllerAdvice nad klasa, czy to jakis nowy sposob o ktorym nie wiem|?

Reply:
Ok